### DISCUSSION TITLE
Magnifier Released

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#64] [Announcements]
[yay or something](https://magnifier.potatophant.net/)

**Comment by FunctionalMetatable** [comment]
yes bois

**Comment by JackyBorderCollie** [comment]
yeaaaaaaaaaaaa boiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii
